image/cli/mascli/functions/gitops_utils

```diff
@@ -282,11 +282,10 @@ function git_lock_branch_name() {
 
   LOCK_NAME=$1
   ACCOUNT_ID=$2
-  REGION_ID=$3
-  CLUSTER_ID=$4
-  MAS_INSTANCE_ID=$5
+  CLUSTER_ID=$3
+  MAS_INSTANCE_ID=$4
 
-  echo -n "lock.${LOCK_NAME}.${ACCOUNT_ID}.${REGION_ID}.${CLUSTER_ID}"
+  echo -n "lock.${ACCOUNT_ID}.${LOCK_NAME}.${CLUSTER_ID}"
```

Comment:
Minor point as I think this would work as-is, but I think I'd prefer to this be:
`"lock.${LOCK_NAME}.${ACCOUNT_ID}.${CLUSTER_ID}"`
to align with how we had it before (minus the region id).

Reply:
changed